tests/agent/python/app.py

```diff
@@ -14,8 +14,7 @@
 
 apm = ElasticAPM(
     app,
-    organization_id='123',
-    app_id='123',
+    app_name='test-app',
     secret_token='123',
```

Comment:
I'm also making it empty. As it is convenient to have it in if you want to test the apm-server with a token.